### ISSUE TITLE
returning a json string or array is a potential security hole, please ensure you really want to do this

### ISSUE THREAD

**Issue by pdehaan** [mozilla-services/readinglist#137]
I was randomly poking around the readinglist server and client and noticed the following output in my server Terminal tab:

```sh
$ make serve

.venv/bin/pserve config/readinglist.ini --reload
Starting subprocess with file monitor
Starting server in PID 23642.
serving on http://0.0.0.0:8000
2015-02-20 10:42:46,544 DEBUG [cliquet][Dummy-2] "GET /" 307 0 (1 ms)
2015-02-20 10:42:46,549 DEBUG [cliquet][Dummy-3] "GET /v0/" 200 129 (2 ms)
2015-02-20 10:42:47,052 DEBUG [cliquet][Dummy-4] "GET /favicon.ico" 307 0 (1 ms)
2015-02-20 10:42:47,097 DEBUG [cliquet][Dummy-5] "GET /v0/favicon.ico" 404 117 (1 ms)
2015-02-20 10:44:21,338 WARNI [cornice][Dummy-2] returning a json string or array is a potential security hole, please ensure you really want to do this.
2015-02-20 10:44:21,338 DEBUG [cliquet][Dummy-2] "OPTIONS /v0/articles" 200 2 (2 ms)
2015-02-20 10:44:21,412 DEBUG [cliquet][Dummy-3] "GET /v0/articles" 401 117 (1 ms)
2015-02-20 10:44:28,165 DEBUG [cliquet][Dummy-4] "GET /v0/fxa-oauth/login" 302 2 (18 ms)
2015-02-20 10:45:56,989 WARNI [cornice][Dummy-5] returning a json string or array is a potential security hole, please ensure you really want to do this.
2015-02-20 10:45:56,989 DEBUG [cliquet][Dummy-5] "OPTIONS /v0/articles" 200 2 (0 ms)
2015-02-20 10:45:57,030 DEBUG [cliquet][Dummy-2] "GET /v0/articles" 401 117 (1 ms)
2015-02-20 10:51:12,551 DEBUG [cliquet][Dummy-3] "GET /v0/fxa-oauth/login" 302 2 (7 ms)
```

Not sure why cornice spews out this warning, but thought I'd file it in case it was interesting to anybody:
> _returning a json string or array is a potential security hole, please ensure you really want to do this._

Pretty sure the "/favicon.ico" and "/v0/favicon.ico" 404s were just me trying to view those pages directly. Not sure that we'd want to add favicons on an app server just to avoid the unlikely event of seeing a bunch of those in the server logs.

**Comment by ametaireau** [comment]
Thanks for reporting that.

We [changed recently this behavior](https://github.com/mozilla-services/cornice/commit/5732a0a88cdc71abd760ed8872c0bc5a67383ab6) in Cornice. I believe we need to do a new release of Cornice with the change in it and update cliquet repo to use this last version.

I'm letting this open until we've done that!

**Comment by Natim** [comment]
It was released in Cornice 0.18
